pkg/cel/ext/kuadrant.go

```diff
@@ -110,3 +111,36 @@ func (l kuadrantLib) ProgramOptions() []cel.ProgramOption {
 func (kuadrantLib) LibraryName() string {
 	return "kuadrant.cel.ext.kuadrant"
 }
+
+func getRegistryWithTypes() *types.Registry {
+	registry, _ := types.NewRegistry(
+		// common.proto
+		&v0.Metadata{},
+		&v0.TargetRef{},
+		&v0.Condition{},
+		&v0.ConditionStatus{},
+
+		// gateway_api.proto
+		&v0.Gateway{},
+		&v0.GatewaySpec{},
+		&v0.Listener{},
+		&v0.GatewayAddresses{},
+		&v0.GatewayStatus{},
+		&v0.ListenerStatus{},
+		&v0.GatewayClass{},
+		&v0.GatewayClassSpec{},
+		&v0.GatewayClassStatus{},
+
+		// policy.proto
+		&v0.Policy{},
+		&v0.PolicyStatus{},
+
+		// kuadrant.proto
+		&v0.PingRequest{},
+		&v0.PongResponse{},
+		&v0.ResolveRequest{},
+		&v0.ResolveResponse{},
+		&v0.Event{},
```

Comment:
There tho, would remain `v0` for now tho!

Reply:
... but I don't think they should be added to this list tho! Why would they be known types to the CEL interpreter?

Reply:
yep, that makes little sense to have them. I'll get rid of them